src/utils/wallet.ts

```diff
@@ -80,25 +60,94 @@ export async function loadPersistedState(walletPath: string, password: string):
 }
 
 export async function loadWalletFromLedger(derivationPath: string): Promise<Identity[]> {
-    const identity = await unlockAddress(0, derivationPath).catch(err => {
+    const identity = await unlockAddress(derivationPath).catch((err) => {
         const errorObj = {
-            name: 'components.messageBar.messages.ledger_not_connect'
+            name: 'components.messageBar.messages.ledger_not_connect',
         };
         console.error('TezosLedgerWallet.unlockAddress', err);
         throw errorObj;
     });
 
-    identity.storeType = StoreType.Hardware;
+    identity.storeType = KeyStoreType.Hardware;
     identity.derivationPath = derivationPath;
     const identities = [identity];
     const localIdentities = getLocalData('identities');
     return prepareToLoad(identities, localIdentities);
 }
 
-export function initLedgerTransport() {
-    TezosLedgerWallet.initLedgerTransport();
+export function loadTokens(network: string) {
+    return knownTokenContracts.filter((token) => token.network === network);
 }
 
-export function loadTokens(network: string) {
-    return knownTokenContracts.filter(token => token.network === network);
+/**
+ * Saves a wallet to a given file.
+ *
+ * @param {string} filename Path to file
+ * @param {Wallet} wallet Wallet object
+ * @param {string} passphrase User-supplied passphrase
+ * @returns {Promise<Wallet>} Wallet object loaded from disk
+ */
+export async function saveWallet(filename: string, wallet: Wallet, passphrase: string): Promise<Wallet> {
```

Comment:
Should these wallet functions not be in ConseilJS? That's where we always had them. Why are we moving them?

Reply:
No, conseiljs is a blockchain data interface library for talking to nodes and indexers. "file wallet"s are clearly a wallet app feature.